docs/reference/security/sts-login.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Secure Token Service (STS) Authentication for lakeFS
+description: Authenticate with lakeFS using Secure Token Service (STS) by leveraging a remote authenticator. This feature enables integration with Identity Providers (IdPs) for secure and efficient user authentication.
+grand_parent: Reference
+parent: Security
+redirect_from:
+  - /reference/remote-authenticator.html
+---
+
+# STS Login
+
+{: .d-inline-block }
+<a style="color: white;" href="#sso-for-lakefs-cloud">lakeFS Cloud</a>
+{: .label .label-green }
+
+{: .d-inline-block }
+<a style="color: white;" href="#sso-for-lakefs-enterprise">lakeFS Enterprise</a>
+{: .label .label-purple }
+
+{: .note}
+> STS Login is available in lakeFS Cloud and lakeFS Enterprise. currently only tested on lakeFS Enterprise.
+
+Secure Token Service (STS) authentication in lakeFS allows users to authenticate via an Identity Provider (IdP), using OIDC Authentication workflow.
+This document outlines the process, from setting up the STS authentication flow to integrating with lakeFS through the [high-level Python SDK](../../integrations/python.md).
+
+## Architecture
```

Comment:
WDYT about moving the architecture section to the end of the page? this way we start with the "how to" and whoever is more interested in the details can continue reading

Reply:
How about before the Sample implementation?

Reply:
Sounds good!